src/types/widgets.ts

```diff
@@ -186,7 +186,7 @@ export type CustomWidgetElementOptions = {
       /**
        * Variable type
        */
-      variableType: 'string' | 'boolean' | 'number'
+      variableType: 'string' | 'boolean' | 'number' | null
```

Comment:
Instead of replicating this same type over and over, could we export the `'string' | 'boolean' | 'number'` from the data-lake file and import it here?